llvm/test/CodeGen/DirectX/sin_error.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: not opt -S -dxil-op-lower %s 2>&1 | FileCheck %s
+
+; This test is expected to fail with the following error
```

Comment:
This comment doesn't really add any information to the check for "LLVM ERROR". Maybe a comment like "DXIL does not support overloads of `sin` for doubles" would be more helpful.

Reply:
Changed.